docs/src/assets/js/scroll-up-btn.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+    const scrollUpBtn = document.getElementById("scroll-up-btn");
+
+    if(window.innerWidth < 1400) {
```

Comment:
I have a suggestion here. This width check can be removed and media query can be used instead to show or hide scroll button for desktop. In `no-js` right now it will show in desktop device too?